src/user-guide/writing-workflows/jinja2.rst

```diff
@@ -348,6 +360,40 @@ will show the workflow with the Jinja2 variables as set.
    be overridden.
 
 
+Using a plugin
+^^^^^^^^^^^^^^
+
+Template plugins such as :ref:`Cylc Rose` should provide a set of template
+variables which can be provided to Cylc. For example, using Cylc Rose you
+add a ``rose-suite.conf`` file which contains a ``[template variables]``
+section which the plugin makes available to Cylc:
+
+For example, with Cylc Rose installed:
+
+.. code-block:: ini
+
+   # rose-suite.conf
+   [template variables]
+   ICP = 1068
+
+.. code-block:: cylc
+
+   #!jinja2
+   [scheduler]
+      allow implicit tasks = True
+   [scheduling]
+      initial cycle point = {{ICP}}
+      [[dependencies]]
+         P1Y = Task1
+
+
+.. code-block:: shell
+
+   > cylc config . -i "[scheduling]initial cycle point"
+   1068
+
```

Comment:
Not sure what I'm doing wrong here @wxtim, hopefully you can spot my mistake...I've tried out this example but hitting an error...I'm on your branches for https://github.com/cylc/cylc-flow/pull/4360 and the cylc-rose sibling, having pip installed them both and I'm getting.... 
![image](https://user-images.githubusercontent.com/37735232/129911100-667164f7-8f8a-401f-b121-50ae395f7a13.png)

Reply:
I don't think that rose supports spaces in `key = value`. Try `ICP=1068`

Reply:
I tried without the spaces :( No luck, same error. My suspicion is it is my environment, I will try on a fresh one and see if that helps.

Reply:
:(